src/_pytest/logging.py

```diff
@@ -141,9 +141,14 @@ def _get_auto_indent(auto_indent_option: Union[int, str, bool, None]) -> int:
 
         if auto_indent_option is None:
             return 0
-        elif type(auto_indent_option) is int:
```

Comment:
mypy doesn't do type narrowing on `type(...) is ..`, so I changed the code to use `isinstance` instead. Then things need to move a bit because `isinstance(True, int)` is true.